server/shared/database/migrations/20210125094759-add-edited_at_column_to_comment.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
```

Comment:
It's never a good practice to change migrations. Since this migration is not faulty, add new migration for hydrating `edited_at` field.